cli/src/common.ts

```diff
@@ -62,6 +62,26 @@ export async function checkPackage(_config: Config): Promise<string | null> {
   return null;
 }
 
+export async function checkCapacitorPackage(
+  config: Config,
+  packageName: string,
+): Promise<string | null> {
+  const pkg = await getCapacitorPackage(config, packageName);
+
+  if (!pkg) {
+    return (
+      `Could not find the ${chalk.bold(
+        packageName,
+      )} platform. Does it need to be installed?\n\n` +
+      `    ${chalk.bold(`npm install @capacitor/${packageName}`)}${chalk.dim(
```

Comment:
We need some way of telling them to install the dependency. A vast majority of people will be able to copy/paste the command to resolve the issue, and the rest can translate the npm commands to their package manager of choice.

Reply:
But a lot of people just use whatever their framework uses, not their choice, if you tell them to npm install, they will npm install despite they might be using other thing.
You can see a lot of people running `ionic cordova plugin add plugin-id` in capacitor projects because they read it in ionic docs that's how you install a Cordova plugin. But that's not really the way of installing Cordova plugins in Capacitor projects.